### ISSUE TITLE
Error on performing "yarn watch"

### ISSUE THREAD

**Issue by jereef** [benwinding/react-admin-firebase#30]
Error in console: 

```
yarn run v1.16.0
$ microbundle watch
node-resolve: setting options.module is deprecated, please override options.mainFields instead
node-resolve: setting options.jsnext is deprecated, please override options.mainFields instead
node-resolve: setting options.module is deprecated, please override options.mainFields instead
node-resolve: setting options.jsnext is deprecated, please override options.mainFields instead
node-resolve: setting options.module is deprecated, please override options.mainFields instead
node-resolve: setting options.jsnext is deprecated, please override options.mainFields instead
Watching source, compiling to dist:
(rpt2 plugin) Error: /home/jeremias/Proyectos/react-admin-firebase/src/providers/database/FirebaseClient.ts(18,5): semantic error TS2741 Property 'clearPersistence' is missing in type 'Firestore' but required in type 'FirebaseFirestore'.
Error: /home/jeremias/Proyectos/react-admin-firebase/src/providers/database/FirebaseClient.ts(18,5): semantic error TS2741 Property 'clearPersistence' is missing in type 'Firestore' but required in type 'FirebaseFirestore'.
    at error (/home/jeremias/Proyectos/react-admin-firebase/node_modules/rollup/dist/rollup.js:3460:30)
    at Object.error (/home/jeremias/Proyectos/react-admin-firebase/node_modules/rollup/dist/rollup.js:21251:17)
    at RollupContext.error (/home/jeremias/Proyectos/react-admin-firebase/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:17186:30)
    at /home/jeremias/Proyectos/react-admin-firebase/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:24943:23
    at arrayEach (/home/jeremias/Proyectos/react-admin-firebase/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:532:11)
    at forEach (/home/jeremias/Proyectos/react-admin-firebase/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:9360:14)
    at printDiagnostics (/home/jeremias/Proyectos/react-admin-firebase/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:24916:5)
    at Object.transform (/home/jeremias/Proyectos/react-admin-firebase/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:26582:17)
    at /home/jeremias/Proyectos/react-admin-firebase/node_modules/rollup/dist/rollup.js:20962:25

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

**Comment by benwinding** [comment]
Thanks for reporting this @jereef,

Looks like it could be a node problem. It's working on my machine using node `v8.15.0`, but failing under `node 10`.

What version of `node` are you using? 

A good utility to try another verison of node is [nvm (Node Version Manager)](https://github.com/nvm-sh/nvm#installation-and-update)

Cheers,
Ben

**Comment by jereef** [comment]
I tried several versions of node and it did not work.I solved this error downgrading the version of firebase-type to 1.2.0.
Hope that helps!

**Comment by benwinding** [comment]
Thanks for the update Jermias!
We might have to tighten up the version constraints on that package. Will
check back on this after the weekend.

**Comment by benwinding** [comment]
Fixed now, by using v1.2.0
